mep/books/queryset.py

```diff
@@ -23,6 +23,7 @@ class WorkSolrQuerySet(AliasedSolrQuerySet):
         'account_start': 'account_start_i',
         'account_end': 'account_end_i',
         'is_uncertain': 'is_uncertain_b',
+        'event_count': 'borrow_count_i',
```

Comment:
isn't this field called `event_count_i`?

Reply:
yes! thanks for catching

Reply:
I may have forgotten it because I have mixed feelings about adding aliases for fields we aren't actually using in the output. Any thoughts?

Reply:
makes sense - i figured if it was important a test would've already caught it. i'm fine not including stuff here until we use it, but weren't we talking about maybe using this one specifically so that there's a visual indicator when you sort by circulation?

Reply:
We're adding to the wishlist a feature to display event count on the search results, so we might want it. I'll leave it in.